wmn-data.json

```diff
@@ -518,6 +518,18 @@
         "cat" : "XXXPORNXXX",
         "valid" : true
        },
+       {
+        "name" : "BabyPips",
+        "uri_check" : "https://forums.babypips.com/u/{account}/summary",
```

Comment:
Can you shift this to `uri_pretty` and make the `https://forums.babypips.com/u/baemax023.json` the `uri_check`? It'll be MUCH easier to grab consistent data from the JSON and then point users at the HTML.